addOns/automation/src/main/java/org/zaproxy/addon/automation/jobs/JobUtils.java

```diff
@@ -625,4 +626,26 @@ public static ScriptWrapper getScriptWrapper(
         }
         return wrapper;
     }
+
+    public static File getFile(String name, AutomationPlan plan) {
+        File planFile = plan.getFile();
+        String nameWithVars = plan.getEnv().replaceVars(name);
+        File f;
+
+        if (planFile != null && !name.startsWith("/")) {
+            f = new File(planFile.getParentFile(), nameWithVars);
+        } else {
+            f = new File(nameWithVars);
+        }
+
+        return f;
+    }
+
+    public static boolean isAbsoluteLiteralPath(String path) {
+        return path.startsWith("/") && !containsVars(path);
+    }
```

Comment:
Careful this isn't windows friendly.

Reply:
Doh! For some reason I thought we would still be using `/` on Windows

Reply:
Well admittedly I haven't tested, but it seems unlikely.

Reply:
Instantiate an actual `Path` from the string and leverage java.nio's https://docs.oracle.com/javase/7/docs/api/java/nio/file/Path.html#isAbsolute()

Reply:
Already done that :D